src/databricks/labs/ucx/installer/workflows.py

```diff
@@ -112,6 +112,112 @@
      f'--parent_run_id=' + dbutils.widgets.get('parent_run_id'))
 """
 
+EXPORT_TO_EXCEL_NOTEBOOK = """
+# Databricks notebook source
+# MAGIC %md
+# MAGIC ##### Exporter of UCX assessment results
+# MAGIC ##### Instructions:
+# MAGIC 1. Execute using an all-purpose cluster with Databricks Runtime 14 or higher.
+# MAGIC 1. Hit **Run all** button and wait for completion.
+# MAGIC 1. Go to the bottom of the notebook and click the Download UCX Results button.
+# MAGIC
+# MAGIC ##### Important:
+# MAGIC Please note that this is only meant to serve as example code.
+# MAGIC
+# MAGIC Example code developed by **Databricks Shared Technical Services team**.
+
+# COMMAND ----------
+
+# DBTITLE 1,Installing Packages
+# MAGIC %pip install {remote_wheel} -q -q -q
+# MAGIC %pip install xlsxwriter -q -q -q
```

Comment:
Why do we suppress the pip install?

```suggestion
# MAGIC %pip install {remote_wheel} -qqq
# MAGIC %pip install xlsxwriter -qqq
```

Note that this will fail for workspaces that have restrictive internet access. To do this similar to how we install ucx, use the `upload_dependencies`, see the `install.py`